### ISSUE TITLE
`forward-port` rich rule can't be applied to a policy with `HOST` ingress zone

### ISSUE THREAD

**Issue by gasinvein** [firewalld/firewalld#1025]
### What happened

I'm trying to use firewall-cmd to redirect outgoing (from the host running firewalld) traffic designated to a specific address to another address/port (this is a common use-case for a transparent proxy).
But trying to add a rich rule with `forward-port` (or an equivalent `--add-forward-port` entry) to a policy object with `--add-ingress-zone HOST` and `--add-egress-zone ANY` results in the following error:
```
Error: "Can't convert policy to chain name: test-out-dnat, nat, False"
```

### What you expected to happen

A `dnat` rule is added to `nat_OUT_` chain for the policy table.

### How to reproduce it (as minimally and precisely as possible)

1. `firewall-cmd --permanent --new-policy test-out-dnat`
2. `firewall-cmd --permanent --policy test-out-dnat --add-ingress-zone HOST`
3. `firewall-cmd --permanent --policy test-out-dnat --add-egress-zone ANY`
4. `firewall-cmd --reload`
5. `firewall-cmd --policy test-out-dnat --add-rich-rule 'rule family=ipv4 destination address=192.168.1.20 forward-port protocol=tcp port=80 to-addr=192.168.2.10 to-port=8080'`
   or
   `firewall-cmd --policy test-out-dnat --add-forward-port 'port=80:proto=tcp:toport=8080:toaddr=192.168.2.10'`

### Anything else we need to know?

This seems similar to #953 and gives the same error.

### Firewalld Version

1.0.5

### Firewalld Backend

nftables

### Linux distribution

Fedora 36

### Linux kernel version

5.19.9-200.fc36.x86_64

**Comment by erig0** [comment]
Hi, this was fixed in 4c8ac0661cb1735a47231b7c1c5d58ced2a5df12 (#903), which is firewalld v1.1.0 and later.

**Comment by gasinvein** [comment]
Great, thanks.
